### PR TITLE
refactor: command cleanup

### PR DESCRIPTION
**Pull request by NathanInnes**
## Description
- Fix a lot of string concat.
- Adjust const naming
- Fixed double space typo

### CODE REVIEW

File: src/commands/moderation/rules.ts [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-import { CommandCategory } from '../../constants';
+import { Channels, CommandCategory, Roles } from '../../constants';
 
 const RULES_EMBED = makeEmbed({
     title: 'FlyByWire Simulations Server Rules',
     description: makeLines([
         'Below are the rules you must follow to participate in this discord server. Failure to abide by these rules could result in a removal from the server. Mute/ban evasions will result in a permanent ban.',
         '',
-        'The <@&739187150909866137> reserve the right to action at discretion.',
+        `The <@${Roles.MODERATION_TEAM}> reserve the right to action at discretion.`,
```
Comment (benw202) [inline src/commands/moderation/rules.ts:10]:
```suggestion
        `The <@&${Roles.MODERATION_TEAM}> reserve the right to action at discretion.`,
```

File: src/constants.ts [hunk 2]
```diff
@@ -21,6 +21,16 @@ export enum Channels {
     BOT_COMMANDS = '902990139670814750',
     BIRTHDAY_CHANNEL = '846470774361161808',
     BIRTHDAY_THREAD = '930923893206679573',
+    A32NX_SUPPORT = '785976111875751956',
```
Comment (benw202) [inline src/constants.ts:24]: If we are adding more here, would it be worth splitting into channels and threads. Or denoting what is a thread by having `_THREAD` like with other entries here?

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/constants.ts (hunk 2 above)
Comment (NathanInnes) [reply src/constants.ts:24]: Yeah should look at that. Will work on later

**Review by benw202 — APPROVED** [review]
LGTM!

**Pull request merged**